src/styles/sidebar/components/share-annotations-panel.scss

```diff
@@ -19,4 +19,12 @@
   &__icon-button {
     @include buttons.button--input;
   }
+
+  // FIXME: Centralize styling of different sizes of Spinner in a reusable
```

Comment:
Note this: spinner sizing is a function of font sizing, as it is in `em`s (which is fine). At some point we should encapsulate some patterns here so that we have the right small set of defined spinner sizes. The spinner at the panel's font size was too small.